### PR TITLE
Navigable items can state if they are relate to implicitly declared items or not.  We want this as we will not display unreference implicitly declared nav items.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
This is part of the Dev15 async find refs feature.

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
Tagging @dotnet/roslyn-ide

**Comment by CyrusNajmabadi** [comment]
Tagging @paulvanbrenk 
This is a simple property addition to an interface.  It will break TS, however, you just need to implement the method by having it return 'true' to maintain all current behavior.

File: src/EditorFeatures/Core/Extensibility/Navigation/INavigableItem.cs [hunk 1]
```diff
@@ -19,9 +19,17 @@ internal interface INavigableItem
         /// </summary>
         bool DisplayFileLocation { get; }
 
+        /// <summary>
+        /// his is intended for symbols that are ordinary symbols in the language sense, and may be
```
Comment (paulvanbrenk) [inline src/EditorFeatures/Core/Extensibility/Navigation/INavigableItem.cs]: +T

**Comment by paulvanbrenk** [comment]
LGTM, thanks for pinging me. (+ @jramsay )

**Comment by balajikris** [comment]
No tests?

**Comment by Pilchie** [comment]
👍

**Pull request merged**